app/_how-tos/kic-proxy-tcp-sni.md

```diff
@@ -142,13 +162,15 @@ In real-world usage, you would create a DNS record for `tls9443.kong.example` po
 ```bash
 echo "hello" | openssl s_client -connect $PROXY_IP:9443 -servername tls9443.kong.example -quiet 2>/dev/null
 ```
+
```

Comment:
Angel and I both ran into issues on validation here (I had him test when it didn't work for me), same error:

```
Connecting to 127.0.0.1
CONNECTED(00000003)
C0FAD3FA01000000:error:0A000126:SSL routines::unexpected eof while reading:ssl/record/rec_layer_s3.c:693:
---
no peer certificate available
---
No client certificate CA names sent
---
SSL handshake has read 0 bytes and written 338 bytes
Verification: OK
---
New, (NONE), Cipher is (NONE)
Protocol: TLSv1.3
This TLS version forbids renegotiation.
Compression: NONE
Expansion: NONE
No ALPN negotiated
Early data was not sent
Verify return code: 0 (ok)
```